modules/terraform-aws-alternat/main.tf

```diff
@@ -40,10 +40,13 @@ locals {
   )
 
   endpoints = merge(local.ec2_endpoint, local.lambda_endpoint)
+
+  reuse_nat_instance_eips = try(length(var.nat_instance_eip_ids), 0) > 0
+  nat_instance_eip_ids    = local.reuse_nat_instance_eips ? var.nat_instance_eip_ids : try(aws_eip.nat_instance_eips[*].id, [])
```

Comment:
What is the purpose of `try` in this statement? Could this be simplified to:

```
  nat_instance_eip_ids    = local.reuse_nat_instance_eips ? var.nat_instance_eip_ids : aws_eip.nat_instance_eips[*].id

```

Reply:
Copypasta'd it from https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/aa61bc4346e1c430df8ec163ae9799d57df4af20/main.tf#L1005 but you're right